### PR TITLE
media-gfx/inkscape: bump to v1.1.1

### PR DESCRIPTION
**Pull request by t0b3**

### CODE REVIEW

File: media-gfx/inkscape/inkscape-1.1.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+inherit cmake flag-o-matic xdg toolchain-funcs python-single-r1
+
+DESCRIPTION="SVG based generic vector-drawing program"
+HOMEPAGE="https://inkscape.org/"
+SRC_URI="https://media.inkscape.org/dl/resources/file/${P}.tar.xz"
+
+LICENSE="GPL-2 LGPL-2.1"
+SLOT="0"
+KEYWORDS="amd64 ~arm ~arm64 ~hppa ~ia64 ppc ppc64 ~riscv ~s390 ~sparc x86"
```
Comment (jonasstein) [inline media-gfx/inkscape/inkscape-1.1.1.ebuild:17]: KEYWORDS need ~ for new ebuilds and after a version bump. Run ekeyword ~all YOUREBUILD

Comment (t0b3) [reply media-gfx/inkscape/inkscape-1.1.1.ebuild]: Thanks for spotting this one

Comment (t0b3) [reply media-gfx/inkscape/inkscape-1.1.1.ebuild]: resolved in new commit

File: dev-cpp/gtkmm/gtkmm-3.24.5.ebuild [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{7..9} )
+PYTHON_COMPAT=( python3_{8..10} )
```
Comment (jonasstein) [inline dev-cpp/gtkmm/gtkmm-3.24.5.ebuild:6]: I think there should be no Py 3.10 changes in this commit, unless it is a strict dependency.

Comment (thesamesam) [reply dev-cpp/gtkmm/gtkmm-3.24.5.ebuild:6]: It's fine if it works and I think they needed to do it to allow Inkscape to get Python 3.10.

Comment (t0b3) [reply dev-cpp/gtkmm/gtkmm-3.24.5.ebuild:6]: Screenshot_20211230_083726
python 3.10 depgraph shows *r+bdep* for inkscape 
source: gentoo python 3.10 qa report https://qa-reports.gentoo.org/output/gpyutils/39-to-310.svg

Comment (t0b3) [reply dev-cpp/gtkmm/gtkmm-3.24.5.ebuild:6]: ... and yes Inkscape and many others work fine on my py310-only box since October.

**Pull request closed without merging**